contrib/dracut/90zfs/zfs-generator.sh.in

```diff
@@ -32,16 +43,23 @@ fi
 root="${root##zfs:}"
 root="${root##ZFS=}"
 
+echo "zfs-generator: writing extension for sysroot.mount to $GENERATOR_DIR"/sysroot.mount.d/zfs-enhancement.conf >> /dev/kmsg
+
 [ -d "$GENERATOR_DIR" ] || mkdir "$GENERATOR_DIR"
-[ -d "$GENERATOR_DIR/sysroot.mount.d" ] || mkdir "$GENERATOR_DIR/sysroot.mount.d"
+[ -d "$GENERATOR_DIR"/sysroot.mount.d ] || mkdir "$GENERATOR_DIR"/sysroot.mount.d
```

Comment:
It was strictly not necessary.  It just looks more consistent with other Dracut code I read.